libcloud/dns/drivers/powerdns.py

```diff
@@ -368,7 +392,7 @@ def list_zones(self):
         response = self.connection.request(action=action, method='GET')
         return self._to_zones(response)
 
-    def update_record(self, record, name, type, data, extra=None):
+    def update_record(self, record, name, type, data, extra=None, disabled=False, comment=None):
```

Comment:
Same here.

Reply:
The new push has been modified: disabled and comment has been added to extra parameter already declared